File: week-02/day-06/src/Loops/Exercises/ParametricAverage.java
import java.util.Scanner;

public class ParametricAverage {
  public static void main(String[] args) {
    // Write a program that asks for a number.
    // It would ask this many times to enter an integer,
    // if all the integers are entered, it should print the sum and average of these
    // integers like:
    //
    // Sum: 22, Average: 4.4

    Scanner scanner = new Scanner(System.in);

    int sum = 0;

    System.out.println("How many Integers: ");
    int numberCount = scanner.nextInt();

    for (int i = 0; i < numberCount; i++) {
      System.out.println("Insert " + (i + 1) + ". Integer: ");
      sum += scanner.nextInt();
    }

    System.out.println("Sum: " + sum + ", Average: " + (float)(sum / numberCount));
  }
}

File: Project/week-21/day-02/Basic JS/Loops/parametric-average.ts
'use strict';

// Write a program that calculates the sum and the average from 1 to a given number.
// Example input: 5
// Example output: Sum: 15, Average: 3

let givenNumber: number = 5;
let sum: number = 0;

for (let i = 1; i <= givenNumber; i++) {
	sum += i;
}

console.log(`Sum: ${sum}, Average: ${sum / givenNumber}`);